inc/crc.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * crc.h
+ *
+ *  Created on: Oct 24, 2018
+ *      Author: piparocj
+ */
+
+// Compiler guard
+#ifndef CRC_H
+#define CRC_H
+
+#include <inttypes.h>
+#include <stdbool.h>
+
+#define	CRC8_ELEMENTS	256					// Number of elements in a CRC8 lookup table
+#define CRC8_MASK		0x80				// CRC8 mask to test if it is the MSB
+#define BITS_PER_BYTE	8
```

Comment:
Again, just a minor formatting thing. We should align the data values if we can

Reply:
Interesting, it doesn't show up like this in Eclipse...

Reply:
I had to use the editor on Github to fix it. Hopefully, it is fixed.